ranger-http-server/pom.xml

```diff
@@ -24,23 +20,12 @@
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-core</artifactId>
-            <version>${dropwizard.version}</version>
         </dependency>
         <dependency>
```

Comment:
We can remove this, dw comes with logback

Reply:
Done.